### ISSUE TITLE
Parameters used to train resnet

### ISSUE THREAD

**Issue by chenchen718** [ppwwyyxx/tensorpack#453]
May I ask about the learning rate, batch size, number of steps in one epoch to obtain the results depicted in the[ figure](https://github.com/ppwwyyxx/tensorpack/tree/master/examples/ResNet) for Resnet 50?

Thanks very much!

**Comment by ppwwyyxx** [comment]
As in the code.

**Comment by chenchen718** [comment]
I am using the specified parameters in the code except for the batchsize ( batchsize for training set is 48 and for validation set is1000 ) to train resnet50 on imagenet. What is confusing me is that up to the 59-th epoch, the train-error-top1 and train-error-top5 are 0.3 and 0.1, but the val-error-top1 and val-error-top5 are still 0.8 and 0.7. Does batchsize influence generalization? What can I do to make validation work better?

**Comment by ppwwyyxx** [comment]
We don't discuss general (non-tensorpack) machine learning questions like this in the issue.
